legacy/src/fabric-custom-xl/web/index.js

```diff
@@ -6,7 +6,6 @@ const DapAssetsRoot = `https://s3-eu-west-1.amazonaws.com/adops-assets/dap-fabri
 const DapAssetsFolder = '[%DapAssetsFolder%]';
 
 const DapAssetsPath = `${DapAssetsRoot}/${DapAssetsFolder}`;
-const ThirdPartyTag = '[%ThirdPartyTag%]';
```

Comment:
Would it be better to just delete the old template in this PR instead of updating it?